ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcMethod.java

```diff
@@ -85,6 +85,7 @@
   ETH_GET_MINER_DATA_BY_BLOCK_HASH("eth_getMinerDataByBlockHash"),
   ETH_GET_MINER_DATA_BY_BLOCK_NUMBER("eth_getMinerDataByBlockNumber"),
   ETH_GET_PROOF("eth_getProof"),
+  ETH_GET_QUORUM_PAYLOAD("eth_getQuorumPayload"),
```

Comment:
I know it's called eth_getQuorum... but would it make more sense for the constant to be GOQUORUM_... and put it with the GQ ones?

Reply:
maybe just like you've done with the class name - GoQuorumEthGetQuorumPayload

Reply:
I think I like this.

Reply:
Done